networkx/algorithms/tests/test_euler.py

```diff
@@ -95,3 +96,57 @@ def test_eulerian_circuit_multigraph(self):
     @raises(nx.NetworkXError)
     def test_not_eulerian(self):    
         f=list(eulerian_circuit(nx.complete_graph(4)))
+
+    def test_eulerian_path(self):
```

Comment:
These should be methods in a new class `class TestEulerianPath(object):`, since they all relate to one particular function that is being tested.